### DISCUSSION TITLE
Accessibility in Flex Insight UI

### DISCUSSION THREAD

**Discussion by ppersky** [twilio-labs/paste#1185] [General] [answered]
I have read and implemented the Inclusive Design Guide from the Paste website, however to ensure I am applying it in the best way I would like your teams perspective and input.

Please evaluate the designs and call out where it falls short, and let me know what you think can be done with reasonable effort. 

[Prototype](https://www.figma.com/proto/0TADSizC9HVgDsUASZmUTT/RT-Queue-Stats-Detail?node-id=197%3A143680&viewport=142%2C1882%2C0.13363584876060486&scaling=min-zoom)

**Comment by richbachman** [comment, marked as answer]
Hi @ppersky. Can you expand a little more about what sort of feedback you're looking for? Did you have specific accessibility questions?

Thanks!

**Reply by corinne-nagel** [reply]
Video can be accessed here: https://twilio.slack.com/archives/CBEQHHDLP/p1613161015092600
